sgt-back-end/index.js

```diff
@@ -0,0 +1,211 @@
+// sudo service postgresql start/status
+// pgweb --d=studentGradeTable
+// node index.js
+// http
+
+const pg = require('pg');
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const express = require('express');
+const app = express();
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Listening on port 3000.');
+});
+
+// RETURNS ALL ROWS FROM "GRADES" TABLE
+
+app.get('/api/grades', (req, res) => {
+  const array = [];
+
+  const sql = `
+    select *
+    from "grades"
+  `;
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      // console.log(grades.length);
+      res.status(200).json(grades);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        array
```

Comment:
I believe you want to send an error object back to the front with a message about an unknown error